packages/salesforcedx-vscode-replay-debugger/tsconfig.json

```diff
@@ -2,9 +2,7 @@
   "compilerOptions": {
```

Comment:
I think a wrong formatter could have been used here. Could you make sure that is formatted via prettier?